tests/go.mod

```diff
@@ -33,13 +33,14 @@ require (
 	pgregory.net/rapid v1.1.0
 )
 
+require cosmossdk.io/collections v0.4.0
```

Comment:
[nit] this `go.mod` looks funny because there is an `// indirect` dependency in the block above. Also I would expect this dep to be included in the block above.

Maybe a `go mod tidy` will resolve? Or manually moving the lines 😅